modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -77,27 +77,27 @@ The `--copy-network` option only copies networking configuration found under `/e
 
 The disk partitions are created on {product-title} cluster nodes during the {op-system-first} installation. Each {op-system} node of a particular architecture uses the same partition layout, unless the default partitioning configuration is overridden. During the {op-system} installation, the size of the root file system is increased to use the remaining available space on the target device.
 
+Kubernetes supports monitoring `/var/lib/containers` when mounted in the same partition as `/var` or a separate partition. 
+
 There are two cases where you might want to override the default partitioning when installing {op-system} on an {product-title} cluster node:
 
-* Creating separate partitions: For greenfield installations on an empty
-disk, you might want to add separate storage to a partition. This is
-officially supported for mounting `/var` or a subdirectory of `/var`, such as `/var/lib/etcd`, on a separate partition, but not both.
+* Creating separate partitions: For greenfield installations on an empty disk, you might want to add separate storage to a partition. For example, mount `/var/lib/containers` in a separate partition, so that kubelet can monitor the `imagefs` directory for container runtimes,`/var/lib/containers` and the `nodefs` directory for root directory files.
 +
 [IMPORTANT]
 ====
-For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
+If you resized your disk size to host a larger file system, consider creating a separate `/var/lib/containers` partition. This is important for a disk formatted to `xfs`, where a high number of allocation groups might cause CPU time issues.
 ====
+
+* Retaining existing partitions: For a brownfield installation where you are reinstalling {product-title} on an existing node and want to retain data partitions installed from your previous operating system, use boot arguments and options with the `coreos-installer` command so that you can keep existing data partitions.
 +
 [IMPORTANT]
 ====
-Kubernetes supports only two file system partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
+The use of a custom partition for mounting ` /var/lib/containers` could result in the partition from not being monitored by {product-title} or alerted on. If you are overriding the default partitioning, see link:https://access.redhat.com/articles/4766521[Understanding OpenShift File System Monitoring (eviction conditions)] for more information about how {product-title} monitors your host file systems.
```

Comment:
That sentence is wrong. The kubelet will always monitor the partition that contains /var/lib/containers. It will always monitor the partition that contains /var/lib/kubelet.

Instead:
```
The use of a custom partition scheme on your node could result some partitions of your node from not being monitored by {product-title} or alerted on. If you are overriding the default partitioning, see link:https://access.redhat.com/articles/4766521[Understanding OpenShift File System Monitoring (eviction conditions)] for more information about how {product-title} monitors your host file systems.
```

Reply:
Thanks, Andreas. Updated.

@djuran . Are you OK with this update?

Reply:
The phrase ` result in some partitions of your node from not being monitored ' doesn't sound right
But I'm not a native-english-speaker, so what do I know

Reply:
@andreaskaris knows this much better than me but from my limited knowledge,  it looks correct